service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -334,6 +359,7 @@ public void markPipelineRunFailed(UUID jobId, String userId) {
    * @return - a PageResponse containing the list of records in the current page and the page tokens
    *     for the next and previous page if applicable
    */
+  @Deprecated
```

Comment:
can we create a ticket to remove this endpoint and all relevant code (including the pagination codue this method calls.  Thanks for these `@Deprecated` annotations!

Reply:
I've just filed [TSPS-622](https://broadworkbench.atlassian.net/browse/TSPS-624), [623](https://broadworkbench.atlassian.net/browse/TSPS-623), and [624](https://broadworkbench.atlassian.net/browse/TSPS-624) which relate to this